docs/index.md

```diff
@@ -23,14 +25,28 @@ terraform {
 }
 
 provider "rhoas" {}
+
```

Comment:
You need to change this in the examples dir or point to a different template

Reply:
Sorry for that. I see it now.. installing terraform documentation tool is not easy :)

Reply:
Ah - I thought it should install automatically. Will look at that.

Reply:
I have fixed that by printing specific version (it looks like that latest version of this command is doing different generation/giving more options